src/components/map/ha-map.ts

```diff
@@ -191,6 +202,11 @@ export class HaMap extends ReactiveElement {
     try {
       [this.leafletMap, this.Leaflet] = await setupLeafletMap(map);
       this._updateMapStyle();
+      this.leafletMap.on("click", (ev) => {
```

Comment:
we need to avoid click firing on double click.
Check here: https://stackoverflow.com/questions/29035896/leaflet-dont-fire-a-click-event-function-on-double-click

Reply:
Do we? I don't see a problem with double click doing the same as single

Reply:
Double click is zoom in. So when you zoom in it sets the pin now. It should just set the pin on single click.